### PR TITLE
Support FxOS Add-ons in multi-search API (bug 1210029)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1210029

### CODE REVIEW

File: mkt/search/views.py [hunk 1]
```diff
@@ -71,37 +74,58 @@ class MultiSearchView(SearchView):
...
+            'index': WebsiteIndexer.get_index()
+        }
+    }, dict)()
+    serializer_classes = {
+        'extension': ESExtensionSerializer,
+        'webapp': ESAppSerializer,
+        'website': ESWebsiteSerializer
+    }
+
+    def get_doc_types_and_indices(self):
```
Comment (ngokevin) [inline mkt/search/views.py]: does this handle the case where doc_type is ```all```?

**Comment by ngokevin** [comment]
flake8

File: mkt/search/views.py (hunk 1 above)
Comment (diox) [inline mkt/search/views.py]: `all` is unsupported to it just uses the default - ` 'webapp,website'`

**Comment by ngokevin** [comment]
r+

**Pull request merged**